part-spec/component.json

```diff
@@ -1,5 +1,5 @@
 {
-	"$id": "https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/component.json",
+	"$id": "https://raw.githubusercontent.com/ajayitem/digital-datasheets/proposed_changes_for_validation/part-spec/component.json",
```

Comment:
this looks like it has your username in it, which is probably not what we want.

I think you want to use something like this instead: https://raw.githubusercontent.com/edatasheets/digital-datasheets/main/part-spec/component.json

Reply:
Yes I agree. I forgot to remove it while doing the validations on my branch since the main wasn't yet up to date with the changes. I will perform the $id change and make another pull request.